Cargo.toml

```
@@ -19,3 +19,11 @@ exclude = [
 ]
 
 [dependencies]
+
+[features]
+# feature only for testing. this feature does not have any effect in final binary
+test-7zip = []
```

Prefix it with underscore to make it clear it isn't actually part of the public API.

Alternatively using `--cfg test_7zip` and `cfg(test_7zip)` also is feasible.

```suggestion
__test-7zip = []
```